randovania/games/fusion/logic_database/Sector 5 ARC.txt

```diff
@@ -18,7 +18,10 @@ Extra - room_id: [0]
   > Door to Nightmare Training Grounds
       All of the following:
           Missiles and Morph Ball
-          Can Climb High Jump Wall or Can Freeze Enemies
+          Any of the following:
+              # Reach the missile block
+              Movement (Beginner) or Can Climb High Jump Wall
```

Comment:
how does the movement work?

Reply:
this was meant to just account for doing a ledge jump to shoot the missile block, since you can't hit it diagonally nhj.
thiking about this more though, this shouldnt be here since you can just use the ladder